### PR TITLE
Add Azure Devops pipeline sample

### PR DESCRIPTION
**Pull request by jfaurskov**
## Description
Adds Azure Devops pipeline sample and related documentation. Replaces #166 

## Changes
- Remove erronous comment from sample Github Action workflow
- Add /azure-pipelines/sample-pipeline.yml
- Add /docs/wiki/Deploy-with-Azure-Pipelines.md
- Add link to /docs/wiki/Deploy-with-Azure-Pipelines.md in /docs/wiki/Customize-Policy-Assignment
- Add links to /docs/wiki/Deploy-with-Azure-Pipelines.md in /docs/wiki/Introduction-to-deploying-ALZ-Monitor
- Add link to /docs/wiki/Deploy-with-Azure-Pipelines.md in /docs/wiki/_Sidebar

## Testing evidence

image

### CODE REVIEW

File: docs/wiki/Deploy-with-Azure-Pipelines.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                    "value": "action@mail.com"
+                }
+            }
+        }
+    }
+}
+```
+
+## 3. Configure and run the pipeline
+First configure your Azure DevOps project with a pipeline hosted in Github as described [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#access-to-github-repositories). The pipeline should be configured to use the [sample-pipeline.yml](../../azure-pipelines/sample-pipeline.yml) file.
```
Comment (paulgrimley) [inline docs/wiki/Deploy-with-Azure-Pipelines.md:58]:
```suggestion
First configure your Azure DevOps project with a pipeline hosted in GitHub as described [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#access-to-github-repositories). The pipeline should be configured to use the [sample-pipeline.yml](../../azure-pipelines/sample-pipeline.yml) file.
```

File: docs/wiki/Deploy-with-Azure-Pipelines.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+- Modify the following values in [sample-pipeline.yml](../../azure-pipelines/sample-pipeline.yml):
+  - Change _Location: "norwayeast"_, to your preferred Azure region
+  - Change _ManagementGroupPrefix: "alz"_, to the pseudo root management group id parenting the identity, management and connectivity management groups.
+  - Change _identityManagementGroup: "alz-platform-identity"_, to the management group for identity in your ALZ implementation.
+  - Change _managementManagementGroup: "alz-platform-management"_, to the management group for management in your ALZ implementation.
+  - Change _connectivityManagementGroup: "alz-platform-connectivity"_, to the management group for connectivity in your ALZ implementation.
+  - Change _LZManagementGroup: "alz-landing-zone"_, to the management group for Landing Zones in your ALZ implementation.
+- Go to Azure DevOps Pipelines and run the pipeline you just created.
+
```
Comment (paulgrimley) [inline docs/wiki/Deploy-with-Azure-Pipelines.md:72]: Are we missing a Service Health parameter? if not can we add a note - we may also need to go back and change the GH action

**Review by paulgrimley — CHANGES_REQUESTED** [review]
I've reviewed the docs, need @Alboroni and / or @pkorolo to review and test pipeline

File: docs/wiki/Deploy-with-Azure-Pipelines.md (hunk 2 above)
Comment (jfaurskov) [reply docs/wiki/Deploy-with-Azure-Pipelines.md:72]: We are not. The service health initiative is assigned at the intermediate root management group level which is set in line 66. I see no reason to add any notes.

**Review by paulgrimley — DISMISSED** [review]
LGTM

**Review by paulgrimley — COMMENTED** [review]
Approved pending technical review of pipeline

**Review by Springstone — APPROVED** [review]
Looks good to me. Reviewed the sample pipeline and it's pretty straightforward - running the same commands as in the deployment wiki.

**Pull request merged**